src/org/etools/j1939_84/bus/RP1210Bus.java

```diff
@@ -308,7 +270,7 @@ private Packet decode(byte[] data, int length) {
      *                    the {@link Packet} to encode
      * @return        a byte array of the encoded packet
      */
-    private byte[] encode(Packet packet) {
+    static public byte[] encode(Packet packet) {
```

Comment:
I assume this is now public as it’s needed in tests?

Reply:
For the traffic generator used in testing.